### PR TITLE
chore: add area dropdown to bug report template

### PR DESCRIPTION
**Pull request by balazsorban44**
This information can be combined with a GitHub action to auto-label issues. See https://github.com/github/issue-labeler.

Waiting with the action until a follow-up PR to see how effective this is.

Also reordered the items in terms of usefulness when triaging.

## Bug

- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Errors have a helpful link attached, see `contributing.md`

## Feature

- [ ] Implements an existing feature request or RFC. Make sure the feature request has been accepted for implementation before opening a PR.
- [ ] Related issues linked using `fixes #number`
- [ ] Integration tests added
- [ ] Documentation added
- [ ] Telemetry added. In case of a feature if it's used or not.
- [ ] Errors have a helpful link attached, see `contributing.md`

## Documentation / Examples

- [ ] Make sure the linting passes by running `pnpm build && pnpm lint`
- [ ] The "examples guidelines" are followed from [our contributing doc](https://github.com/vercel/next.js/blob/canary/contributing/examples/adding-examples.md)

### CODE REVIEW

**Comment by ijjk** [comment]
## Failing test suites 
Commit: 30a9835a1c95d32b9eb87a5cbf3242ebf2d9840c

`pnpm testheadless test/e2e/middleware-general/test/index.test.ts` 
- Middleware Runtime > with i18n > should rewrite correctly for non-SSG/SSP page

<details>
<summary>Expand output</summary>

  ● Middleware Runtime › with i18n › should rewrite correctly for non-SSG/SSP page

    TIMED OUT: success

    <head><meta charset="utf-8"><meta name="viewport" content="width=device-width"><meta name="next-head-count" content="2"><noscript data-n-css=""></noscript><script defer="" nomodule="" src="/_next/static/chunks/polyfills.js?ts=1669113375554"></script><script src="/_next/static/chunks/webpack.js?ts=1669113375554" defer=""></script><script src="/_next/static/chunks/main.js?ts=1669113375554" defer=""></script><script src="/_next/static/chunks/pages/_app.js?ts=1669113375554" defer=""></script><script src="/_next/static/chunks/pages/ssg/%5Bslug%5D.js?ts=1669113375554" defer=""></script><script src="/_next/static/development/_buildManifest.js?ts=1669113375554" defer=""></script><script src="/_next/static/development/_ssgManifest.js?ts=1669113375554" defer=""></script><noscript id="__next_css__DO_NOT_USE__"></noscript></head><body><div id="__next"><p id="ssg">/blog/[slug]</p><p id="query">{"slug":"first"}</p><p id="pathname">/ssg/[slug]</p><p id="as-path">/</p><p id="props">{"now":1669113375555,"params":{"slug":"first"}}</p></div><script src="/_next/static/chunks/react-refresh.js?ts=1669113375554"></script><script id="__NEXT_DATA__" type="application/json">{"props":{"pageProps":{"now":1669113375555,"params":{"slug":"first"}},"__N_SSG":true},"page":"/ssg/[slug]","query":{"slug":"first"},"buildId":"development","isFallback":false,"gsp":true,"locale":"en","locales":["en","fr","nl"],"defaultLocale":"en","scriptLoader":[]}</script><div id="__next-build-watcher" style="position: fixed; bottom: 10px; right: 20px; width: 0px; height: 0px; z-index: 99999;"></div><next-route-announcer><p aria-live="assertive" id="__next-route-announcer__" role="alert" style="border: 0px; clip: rect(0px, 0px, 0px, 0px); height: 1px; margin: -1px; overflow: hidden; padding: 0px; position: absolute; width: 1px; white-space: nowrap; overflow-wrap: normal;">/</p></next-route-announcer><script src="/_next/static/chunks/pages/about/a.js"></script></body>

      532 |
      533 |   if (hardError) {
    > 534 |     throw new Error('TIMED OUT: ' + regex + '\n\n' + content)
          |           ^
      535 |   }
      536 |   return false
      537 | }

      at check (lib/next-test-utils.js:534:11)
      at Object.<anonymous> (e2e/middleware-general/test/index.test.ts:413:7)

</details>


Read more about building and testing Next.js in [contributing.md](https://github.com/vercel/next.js/blob/canary/contributing.md).

File: .github/ISSUE_TEMPLATE/1.bug_report.yml [hunk 1]
```diff
@@ -21,36 +21,52 @@ body:
...
+        - CLI (create-next-app)
+        - Data fetching (gS(S)P, getInitialProps)
+        - Dynamic imports (next/dynamic)
+        - ESLint (eslint-config-next)
+        - Font optimization (@next/font)
+        - Internationalzation (i18n)
+        - Image optmization (next/image, next/legacy/image)
+        - Jest (next/jest)
+        - Middleware / Edge (API routes, runtime)
+        - Package manager (pnpm, Yarn PnP)
```
Comment (styfle) [inline .github/ISSUE_TEMPLATE/1.bug_report.yml:39]:
```suggestion
        - Package manager (npm, pnpm, Yarn)
```

File: .github/ISSUE_TEMPLATE/1.bug_report.yml [hunk 2]
```diff
@@ -21,36 +21,52 @@ body:
...
+        - Routing (next/router, next/navigation, Link)
```
Comment (styfle) [inline .github/ISSUE_TEMPLATE/1.bug_report.yml:40]:
```suggestion
        - Routing (next/router, next/navigation, next/link)
```

File: .github/ISSUE_TEMPLATE/1.bug_report.yml [hunk 3]
```diff
@@ -21,36 +21,52 @@ body:
...
+        - Script optimizatzion (next/script)
+        - 'Standalone mode (output: "standalone")'
+        - SWC minifier (swcMinify: true)
+        - SWC transpilation
+        - 'Turbopack (--turbo)'
+        - TypeScript
+        - Other
```
Comment (styfle) [inline .github/ISSUE_TEMPLATE/1.bug_report.yml:47]: How about `next export`?

File: .github/ISSUE_TEMPLATE/1.bug_report.yml [hunk 4]
```diff
@@ -21,36 +21,53 @@ body:
...
       required: true
-  - type: input
+  - type: dropdown
     attributes:
-      label: What browser are you using? (if relevant)
-      description: 'Please specify the exact version. For example: Chrome 100.0.4878.0'
+      label: Which area of Next.js is affected? (leave empty if unsure)
+      multiple: true
+      options:
+        - App directory (appDir: true)
```
Comment (styfle) [inline .github/ISSUE_TEMPLATE/1.bug_report.yml:29]: Does this need to be in quotes?

File: .github/ISSUE_TEMPLATE/1.bug_report.yml [hunk 5]
```diff
@@ -21,36 +21,53 @@ body:
...
+        - Internationalzation (i18n)
+        - Image optmization (next/image, next/legacy/image)
+        - Jest (next/jest)
+        - Middleware / Edge (API routes, runtime)
+        - Package manager (npm, pnpm, Yarn)
+        - Routing (next/router, next/navigation, next/link)
+        - Script optimizatzion (next/script)
+        - 'Standalone mode (output: "standalone")'
+        - Static HTML Export (next export)
+        - SWC minifier (swcMinify: true)
```
Comment (styfle) [inline .github/ISSUE_TEMPLATE/1.bug_report.yml:44]: Same here. Missing quotes? 

Idk yaml ¯\_(ツ)_/¯

**Comment by balazsorban44** [comment]
`'` all the things!

File: .github/ISSUE_TEMPLATE/1.bug_report.yml [hunk 6]
```diff
@@ -62,3 +79,11 @@ body:
   - type: markdown
     attributes:
       value: These steps are used to add integration tests to ensure the same issue does not happen again. Thanks in advance!
+  - type: input
+    attributes:
+      label: What browser are you using? (if relevant)
```
Comment (styfle) [inline .github/ISSUE_TEMPLATE/1.bug_report.yml:84]:
```suggestion
      label: Which browser are you using? (if relevant)
```

**Review by styfle — APPROVED** [review]

**Pull request merged**